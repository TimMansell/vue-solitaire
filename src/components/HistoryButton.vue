<template>
  <Button
    :is-stacked="true"
    :check-disabled="true"
    to-page="history"
    data-test="history-btn"
  >
    Games
  </Button>
</template>

<script>
import Button from './Button.vue';

export default {
  name: 'HistoryButton',
  components: {
    Button,
  },
};
</script>
