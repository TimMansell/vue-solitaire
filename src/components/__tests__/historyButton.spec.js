import { shallowMount } from '@vue/test-utils';
import HistoryButton from '@/components/HistoryButton.vue';

describe('HistoryButton.vue', () => {
  it('matches snapshot', () => {
    const wrapper = shallowMount(HistoryButton);

    expect(wrapper).toMatchSnapshot();
  });
});
