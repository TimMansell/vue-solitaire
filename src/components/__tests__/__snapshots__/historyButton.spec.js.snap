// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HistoryButton.vue matches snapshot 1`] = `
<button-stub type="default" isstacked="true" checkdisabled="true" size="md" topage="history" data-test="history-btn">
  Games
</button-stub>
`;
